framework/db/mssql/QueryBuilder.php

```diff
@@ -35,7 +35,7 @@ class QueryBuilder extends \yii\db\QueryBuilder
         Schema::TYPE_BIGINT => 'bigint',
         Schema::TYPE_FLOAT => 'float',
         Schema::TYPE_DOUBLE => 'float',
-        Schema::TYPE_DECIMAL => 'decimal',
+        Schema::TYPE_DECIMAL => 'decimal(18,0)',
```

Comment:
> @dynasource this change for mssql only and (18, 0) is default values for SQL Server.
> Other DBMS have own type mapping

This is correct, entries in these fields should always use the default by the dbms.